src/main/frontend/modules/shortcut/config.cljs

```diff
@@ -423,14 +427,33 @@
                                      :inactive (not (config/plugin-config-enabled?))
                                      :fn       plugin-config-handler/open-replace-plugins-modal}
 
+   :ui/clear-all-notifications      {:binding false
+                                     :fn      :frontend.handler.notification/clear-all!}
+
    :editor/toggle-open-blocks       {:binding "t o"
                                      :fn      editor-handler/toggle-open!}
 
    :ui/toggle-cards                 {:binding "t c"
                                      :fn      ui-handler/toggle-cards!}
 
    :git/commit                      {:binding "mod+g c"
-                                     :fn      commit/show-commit-modal!}})
+                                     :fn      commit/show-commit-modal!}
+
+   :dev/show-block-data            {:binding false
+                                    :inactive (not (state/developer-mode?))
+                                    :fn :frontend.handler.common.developer/show-block-data}
```

Comment:
Having keyword fns is helpful for two reasons. It allows us to datafy our commands which will be valuable later to allow users more command functionality e.g. create new commands in their config. It also it allows commands to call namespaces that previously created circular dependencies. The developer ns had a circular dependency with frontend.ui and shortcut.config. Rather than rely on the anti-pattern of making events in frontend.handler.events for every fn call, we can now do this